packages/connect/src/impl/core-in-module.ts

```diff
@@ -0,0 +1,272 @@
+import EventEmitter from 'events';
```

Comment:
this new impl should be implemented in 

`connect/src/index`

and also it can be used in connect-web package that is implemented in suite-web to get rid of iframe.